hotshot-state-prover/src/mock_ledger.rs

```diff
@@ -48,6 +48,9 @@ type SchnorrSignKey = jf_signature::schnorr::SignKey<ark_ed_on_bn254::Fr>;
 /// Stake table capacity used for testing
 pub const STAKE_TABLE_CAPACITY: usize = 10;
 
+/// Number of blocks per epoch for testing
+pub const BLOCKS_PER_EPOCH: u32 = 3;
+
```

Comment:
can we set to `u32::MAX` for now?

Reply:
since the blocks_per_epoch is used to determine how many states to generate, which is then used in the tests, I would say we need to choose a small number. I would opt for 5 though

Reply:
okay, sounds good.